.github/CODEOWNERS

```diff
@@ -482,6 +482,7 @@
 # PRLabel: %Monitor - Exporter
 # ServiceLabel: %Monitor - Exporter %Service Attention
 /sdk/monitor/Azure.Monitor.OpenTelemetry.*/     @cijothomas @reyang @rajkumar-rangaraj @TimothyMothra @vishweshbankwar
+/sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/     @cijothomas @reyang @rajkumar-rangaraj @TimothyMothra @vishweshbankwar
```

Comment:
I assume we'll want to use the same "Exporter" labels because this is the same team and essentially the same product. 

Please comment if this needs to be different.

Reply:
Let us have a different label name. It is easier for maintenance.

Reply:
I named the new label "Monitor - Distro" for now.